### DISCUSSION TITLE
Add developer contributing info in README?

### DISCUSSION THREAD

**Discussion by timClicks** [stencila/stencila#872] [Ideas] [answered]
Hi there,

I am looking into the process of building `stencila` from scratch. Here are the steps that I've taken:

1. ensure Rust is installed (https://rustup.rs)
2. (create GitHub fork &&) clone stencila repo
3. `cd stencila/rust`
4. `make setup`
5. `make build`

Is that correct? Is that it? Would it be worthwhile to add these instructions to the README?

**Comment by nokome** [comment]
Hi Tim, thanks for stopping by here.

Yes, those should be the minimum steps needed to build a binary from scratch. I'd appreciate your thoughts on how those steps could be improved. And yes, they should be on the README somewhere.

**Comment by timClicks** [comment]
Awesome :)

Those steps worked for me, by the way. The build process was pretty
seamless. I was just worried that I was missing something - it looked too
easy. I did get tripped at one point by running `make build` from the root
directory and then needing a few cargo plugins, but looked into the
rust/Makefile and saw the setup make target.

**Comment by nokome** [comment, marked as answer]
I have added a "Develop" section to a new, CLI-specific README: https://github.com/stencila/stencila/tree/master/cli#readme. Similar developer guides will be added to the other subfolders.
